website/source/docs/platform/k8s/service-sync.html.md

```diff
@@ -67,17 +67,17 @@ sync to understand how the syncing works.
 The sync process must authenticate to both Kubernetes and Consul to read
 and write services.
 
-For Consul, the process accepts both the standard CLI flag `-token` and
-the environment variable `CONSUL_HTTP_TOKEN`. This should be set to an
-Consul [ACL token](/docs/guides/acl.html) if ACLs are enabled. This
-can also be configured using the Helm chart to read from a Kubernetes
-secret.
-
 For Kubernetes, a valid kubeconfig file must be provided with cluster
 and auth information. The sync process will look into the default locations
 for both in-cluster and out-of-cluster authentication. If `kubectl` works,
 then the sync program should work.
 
+For Consul, if ACLs are configured on the cluster, a Consul [ACL token](/docs/guides/acl.html)
+will need to be provided. The process accepts this token by using the
+`CONSUL_HTTP_TOKEN` environment variable. This token should be set as a
+[Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/#creating-your-own-secrets)
```

Comment:
As a rough example, would that process look like this:

```
kubectl create secret CONSUL_HTTP_TOKEN <your-token-here>
```
If yes, then maybe its worth noting you would only want to set it once and Kubernetes will distribute it for you to all Consul agents.

Reply:
So you will end up using `kubectl create secret`, but came name it however you would like. Reference to that secret should happen in the Helm chart so that the token can be attached to the sync process. A note: this is _not_ a client token, it's a token for the catalog sync pod that runs `consul-k8s`. Client tokens would be dealt with manually by connecting to the Consul cluster, so should be covered by the general ACL documentation.

Reply:
Do you think its obvious that users should not use a client token? Maybe add a note here for now, then we can update the ACL docs later?